common/rfb/CMsgReader.cxx

```diff
@@ -205,8 +205,8 @@ void CMsgReader::readSetXCursor(int width, int height, const Point& hotspot)
   if (width > maxCursorSize || height > maxCursorSize)
     throw Exception("Too big cursor");
 
-  rdr::U8 pr, pg, pb;
-  rdr::U8 sr, sg, sb;
+  rdr::U8 pr = 0, pg = 0, pb = 0;
+  rdr::U8 sr = 0, sg = 0, sb = 0;
```

Comment:
I'm not a huge fan of this as it basically just turns off the warning and might hide future problems. Better to restructure things. You should be able to move the following section in to the if clause. That would also allow you to rescope those variables.